Sketch.java

```diff
@@ -24,13 +26,74 @@ public void setup() {
    */
   public void draw() {
 	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+	drawFace(200,40);
+  drawFace2(100,50, 56, 255);
+  getLocation(width/2, height/2);
```

Comment:
good effort in defining a method with a return value, however, the point of the return value is that you do something with the value that is returned.